packages/babel-plugin-proposal-class-properties/src/index.js

```diff
@@ -83,6 +83,18 @@ export default declare((api, options) => {
     },
   };
 
+  // Traverses the class scope, handling private static name references.
+  const staticPrivatePropertyVisitor = {
```

Comment:
We should be able to reuse `privateNameVisitor`. This is necessary because we have to use its inner traverser, too.

Reply:
Ok, I kept them separate to make sure I don't mess with your logic but indeed, it's better to change just a few things in your logic than to branch out an entirely new one!